lib/Transforms/YosysOptimizer/YosysOptimizer.cpp

```diff
@@ -121,8 +132,10 @@ Value convertIntegerValue(Value value, Type convertedType, OpBuilder &b,
   return allocOp.getResult();
 }
 
-/// Convert a secret.generic's operands secret.secret<i3>
-/// to secret.secret<memref<3xi1>>.
+/// Convert a secret.generic's operands:
+///
+///  - secret.secret<i3> -> secret.secret<tensor<3xi1>>
+///  - secret.secret<memref<Nxi3>> -> secret.secret<memref<3Nxi1>>
```

Comment:
are these accurate? i think we generally are converting `i3` to `memref<3xi1>` and `memref<Nxi3>` to `memref<Nx3xi1>`

Reply:
Will revert. I'm not sure why I wrote this.